Example/SnabblePayExample/Views/CardView.swift

```diff
@@ -60,28 +60,32 @@ struct CardView: View {
                 .padding([.top])
                 .frame(width: toggleSize ? 160 : 60)
             Spacer()
-            
-            VStack(alignment: .leading, spacing: 6) {
+            VStack(alignment: .leading, spacing: 8) {
+                Text(model.ibanString)
+                    .font(.headline)
```

Comment:
Warum hast du den Menlo-Font gegen .headline getauscht? In der Story gehts doch nur um die geänderte Reihenfolge. Für iOS hatten wir damals gesagt, das der Font für die IBAN okay ist.

Reply:
Weil es im Design so ist, kann es auch wieder ändern, aber das sieht vollkommen anders aus als der Ausschnitt aus dem Issue.

Ich habe mit Jessi geklärt das es ok ist das der Text weiß ist.